src/services/mongoose/schema.types.d.ts

```diff
@@ -110,8 +106,23 @@ interface UserRecord {
   title?: string
   coordinates?: CoordinateObjectForUser
 
+  // TODO: delete
```

Comment:
Pretty unspescific comment... when does it get deleted?

Reply:
when this second migration has been run: https://github.com/GaloyMoney/galoy/pull/1861/files/33b17223db559cb9b1dc8d5e49df8d7b53cd5ec3